frontend/src/components/Calendar/calendar.css

```diff
@@ -0,0 +1,183 @@
+* {
+    font-family: 'Montserrat', sans-serif;
+    font-style: normal;
+}
+
+.calendar {
+    width: 900px;
+    height: 600px;
+    display: flex;
+    flex-direction: column;
+    margin-top: 2rem;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #ffffff;
+
+}
+
+.calendar-header {
+    width: 100%;
+    height: 100px;
+    margin: auto;
+    display: flex;
+    align-items: left;
+    justify-content: space-between;
+    font-style: bold;
+
+
+}
+
+.title {
+    width: 25%;
+    height: 100%;
+    display: flex;
+    align-items: center;
+}
+
+.title h2 {
+    margin: auto;
+}
+
+.tools {
+    width: 23%;
+    height: 150%;
+    display: flex;
+    align-items: center;
+}
+
+.tools button {
+    float: left;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+    background-color: #ffffff;
+    border: .5px solid #d9d9d9;
+}
+
+.tools button span:hover {
+    color: #99cccc;
+}
+
+.today-button {
+    background-color: white;
+    color: black;
+    border: .5px solid #e9e9e9;
```

Comment:
if i remember correctly, it's impossible for something to be less than 1 px. You can get the effect of less than 1px though with colors

Reply:
https://stackoverflow.com/a/13891295